packages/util/src/signature.ts

```diff
@@ -4,10 +4,23 @@ import { toBuffer, setLengthLeft, bufferToHex, bufferToInt, bufferToBigInt } fro
 import { SECP256K1_ORDER, SECP256K1_ORDER_DIV_2 } from './constants'
 import { assertIsBuffer } from './helpers'
 
+/**
+ * Interface for ECDSA signature
+ *
+ * @interface ECDSASignature
+ *
```

Comment:
Ah, I actually only meant the `@interface ECDSASignature` part, where TypeScript already knows a) that it's an interface and b) the name, so didn't mean the text comment itself.

Rather a bit of a nit here, since the additional information in the text is also not that big, so guess we can keep it this way.